app/components/Icon.tsx

```diff
@@ -0,0 +1,17 @@
+import type { SVGProps } from "react";
+
+export interface IconProps extends SVGProps<SVGSVGElement> {
+    name: string;
+}
+
+// TODO: Make this type-safe based on all of the <symbol id>s in /images/icons.svg
+// How? No clue...
+export function Icon({ name, viewBox = "0 0 64 64", ...props }: IconProps) {
```

Comment:
Some of the icons look to be a little bit big, like Twitter and LinkedIn for example. Not sure if it's a generic problem, or just certain ones.